### ISSUE TITLE
Better compatibility for enhancement rom hacks

### ISSUE THREAD

**Issue by EK-Gras** [adrienntindall/hgss-map-randomizer#83]
I personally prefer to play the "Storm Silver" ROMhack for Pokemon Soul Silver which adds various enhancements, most notably making it possible to complete the Pokedex via added NPCs and revamped pokemon encounter tables for routes. I only request that compatibility for ROM hacks such as Storm Silver be added because Storm Silver previously was compatible on an older version which now seems to be unavailable (most likely the initial release). I suspect that due to added features in later versions, the randomizer overwrites added/changed events in ROM hacks, leading to buggy behavior (for example, Storm Silver adds a conversation with Cynthia outside of Mr. Pokemon's house at the beginning of the game, but if one attempts to randomize the patched ROM, Cynthia can no longer be interacted with and simply blocks the entrance). This may be outside of the scope of this project, and since my knowledge of ROM hacking is limited I am unsure as to what difficulties adding this capability may entail, but I figured I'd at least bring it up.

**Comment by adrienntindall** [comment]
Yup, in order to fix a ton of bugs related to sequence breaking and to add the ability to have the e4 rooms/certain other maps like the rocket hideout need to be able to be included in the pool in the first place, the changes are required. Other maps include QoL changes that allow for the national dex map to be in early too, which is also required since it's otherwise very buggy. You're welcome to try and change the scr_seq.narc file to fix the issues, though Im not 100% sure what would break as a result of doing that

**Comment by EK-Gras** [comment]
Thanks for your reply. Would you happen to perhaps know which version I could've been using that worked with the patched ROM? If nothing else I'd like to at least play on an older version that somewhat works for what I'm trying to do, but the oldest downloadable release (1.2.7 if I'm not mistaken) doesn't seem to function properly either.

**Comment by adrienntindall** [comment]
Some of the older versions had code I didn't intend to release to the public for various reasons, so I can't give you an older version sorry
